macros/geo/haversine_distance.sql

```diff
@@ -3,17 +3,48 @@ This calculates the distance between two sets of latitude and longitude.
 The formula is from the following blog post:
 http://daynebatten.com/2015/09/latitude-longitude-distance-sql/
 
-The arguments should be float type. 
+The arguments should be float type.
 #}
 
-{% macro haversine_distance(lat1,lon1,lat2,lon2) -%}
-    {{ return(adapter.dispatch('haversine_distance', packages = dbt_utils._get_utils_namespaces())(lat1,lon1,lat2,lon2)) }}
+{% macro degrees_to_radians(degrees) -%}
+    acos(-1) * {{degrees}} / 180
+{%- endmacro %}
+
+{% macro haversine_distance(lat1,lon1,lat2,lon2,unit='mi') -%}
+    {{ return(adapter.dispatch('haversine_distance', packages = dbt_utils._get_utils_namespaces())(lat1,lon1,lat2,lon2,unit)) }}
 {% endmacro %}
 
-{% macro default__haversine_distance(lat1,lon1,lat2,lon2) -%}
+{% macro default__haversine_distance(lat1,lon1,lat2,lon2,unit) -%}
```

Comment:
```suggestion
{% macro default__haversine_distance(lat1, lon1, lat2, lon2, unit='mi') -%}
```

Reply:
done, wasn't sure if we wanted to implement it on the specific macros but that makes sense.